This patch allows the 'asprintf' declaration to be visible, by
including <config.h>, which defines _GNU_SOURCE.


--- xmodmap-1.0.7/xmodmap.c	2013-11-25 23:25:25.000000000 +0100
+++ xmodmap-1.0.7/xmodmap.c	2013-11-25 23:25:27.000000000 +0100
@@ -26,6 +26,7 @@ from The Open Group.
 
 */
 
+#include <config.h>
 #include <X11/Xos.h>
 #include <X11/Xlib.h>
 #include <stdio.h>
